test/cpp/jit/test_lite_interpreter.cpp

```diff
@@ -827,6 +946,60 @@ static auto reg =
               return c10::make_intrusive<TorchBindLiteInterpreterTestStruct>();
             });
 
+// This test JIT backend is intended to do the minimal amount of work
+// necessary to test that the JIT backend registration endpoints and
+// code generation are working correctly. It is not intended to
+// produce numerically correct results.
+class MulTestBackend : public PyTorchBackendInterface {
+ public:
+  // Constructor.
+  explicit MulTestBackend() {}
+  virtual ~MulTestBackend() = default;
+
+  c10::IValue preprocess(
+      c10::IValue mod,
+      c10::impl::GenericDict method_compile_spec) override {
+    return mod;
+  }
+
+  c10::impl::GenericDict compile(
+      c10::IValue processed,
+      c10::impl::GenericDict method_compile_spec) override {
+    auto spec =
+        c10::impl::toTypedDict<std::string, at::IValue>(method_compile_spec);
+
+    // Return the same string as a value for every key in method_compile_spec.
+    auto handles = c10::Dict<std::string, std::string>();
+    for (const auto& it : spec) {
+      handles.insert(it.key(), it.key());
+    }
+    return c10::impl::toGenericDict(handles);
+  }
+
+  c10::impl::GenericList execute(
+      c10::IValue handle,
+      c10::impl::GenericList inputs) override {
+    TORCH_INTERNAL_ASSERT(handle.isString());
+    TORCH_INTERNAL_ASSERT(inputs.size() > 0);
+
+    c10::List<at::Tensor> output_list;
+
+    c10::IValue value = inputs[0];
+    at::Tensor x = value.toTensor();
+    x = x.clone();
+    at::Tensor output = value.toTensor();
+    output = output.clone();
+
+    if (handle.toStringRef() == "forward") {
```

Comment:
So this is somewhat decoupled from what the module under test defines in ToBackend. I would suggest to do this if possible:
1. In preprocess just "compile" forward method of the module by lowering it to bytecode. Using `getFunctionTuple` defined in export_module.cpp. This way we get the bytecode corresponding to that method and that can be serialized as part of the module?. Output of the `getFunctionTuple` can be stored as a new attribute of the returned module such as `compiled_methods`.
2. Then in `compile` part of the backend, extract `compiled_methods` attribute and then construct `Code` out of it the way we do in lite interpreter.

This probably is little too long of a path but thought it might showcase an example of how to serialize compiled methods.